ProStudio/ProStudio/Classes/Screens/ProjectDiscussion/CustomCells/ButtonCell/ButtonCell.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  ButtonCell.swift
+//  ProStudio
+//
+//  Created by Zimma on 10/10/2018.
+//  Copyright © 2018 Nikita. All rights reserved.
+//
+
+import UIKit
+
+class ButtonCell: UITableViewCell {
+
+    @IBOutlet weak var button: CustomButton!
+    var delegat: ButtonCellDelegat?
```

Comment:
we don't using delegates for this cases. 

Here, you need to use completion block. Because, it's not difference to use delegates or button targets.